holoviews/util/locator.py

```diff
@@ -0,0 +1,171 @@
+"""
+Minimal set of functionality of Matplotlib's MaxNLocator to choose contour
+levels without having to have Matplotlib installed.
```

Comment:
Maybe add the Matplotlib version that was used for inspiration?

Reply:
Yes.